.env

```diff
@@ -46,3 +46,7 @@ IP_HASH_SALT='zout'
 INFO_SITE_URL='https://u-can-act.nl'
 
 PUSH_SUBSCRIPTION_URL='http://web:3000/api/v1/data/create_raw'
+BASE_PLATFORM_URL='http://localhost:3000'
+
+SHARED_SECRET='shared secret for generating email invite tokens'
+REGISTRATION_URL='http://localhost:3001/email-registratie'
```

Comment:
this is the url for the questionnaire platform? (:3001?)

Reply:
Yep. this is the quest-front-end stuff.